gno_github_agent/.env

```diff
@@ -0,0 +1,5 @@
+GNO_MNEMONIC=source bonus chronic canvas draft south burst lottery vacant surface solve popular case indicate oppose farm nothing bullet exhibit title speed wink action roast
```

Comment:
Is this normal that we keep this mnemonic here ?

Reply:
fixed on f4a2a1f469fa31fc20f1534f07e68ae1d7408f89